pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -1674,15 +1866,56 @@ func withOCIBundles(cm *corev1.ConfigMap) *corev1.ConfigMap {
 	return new
 }
 
-func TestReconcileOnCancelledPipelineRun(t *testing.T) {
+func withEmbeddedStatus(cm *corev1.ConfigMap, flagVal string) *corev1.ConfigMap {
+	newCM := cm.DeepCopy()
+	newCM.Data[embeddedStatusFeatureFlag] = flagVal
+	return newCM
+}
+
+func TestReconcileOnCancelledPipelineRunFullEmbeddedStatus(t *testing.T) {
```

Comment:
This test case could likely also use the pattern you're establishing (parameterizing the embedded status), because it doesn't really check any parts of the child statuses

Reply:
Yeah, good catch.